src/game/models/mod.rs

```diff
@@ -2,156 +2,16 @@
 
 use std::f64;
```

Comment:
Weird, this program does warn. I wonder if the import checker more globally to your crate?

```rust
use std::f64;
use std::f64::consts::PI;

fn main() {
    println!("PI {}", PI);
}
```